integration/microservices/README.md

```diff
@@ -1,35 +1,39 @@
 # tempo-load-test
```

Comment:
I used `prettier` to format the file.  How do we feel about the markdown style change here?

Reply:
Possibly useful if we built it into CI, but I don't think it's a big deal.